plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -312,4 +352,14 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
       deactivateLocationComponent()
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * The default puck 2D style without showing bearing.
+ */
+lateinit var PUCK_2D_DEFAULT: LocationPuck2D
+
+/**
+ * The puck 2D style with bearing show.
+ */
+lateinit var PUCK_2D_SHOW_BEARING: LocationPuck2D
```

Comment:
@Chaoba why can't we use generated `PuckBearingSource` from `ConfigProperties.kt`? This solution with 2 lateinit vars located not even within a class looks imho a bit strange